core/src/main/scala/cats/data/IorT.scala

```diff
@@ -439,6 +439,60 @@ abstract private[data] class IorTInstances extends IorTInstances1 {
       type F[x] = IorT[P.F, E, x]
       type Dummy // fix to make this one more specific than the catsDataParallelForIorTWithSequentialEffect, see https://github.com/typelevel/cats/pull/2335#issuecomment-408249775
 
+      val parallel: IorT[M, E, *] ~> IorT[P.F, E, *] =
+        new (IorT[M, E, *] ~> IorT[P.F, E, *]) {
+          def apply[A](fm: IorT[M, E, A]): IorT[P.F, E, A] = IorT(P.parallel(fm.value))
+        }
+      val sequential: IorT[P.F, E, *] ~> IorT[M, E, *] =
+        new (IorT[P.F, E, *] ~> IorT[M, E, *]) {
+          def apply[A](ff: IorT[P.F, E, A]): IorT[M, E, A] = IorT(P.sequential(ff.value))
+        }
+
+      private[this] val FA: Applicative[P.F] = P.applicative
+      private[this] val IorA: Applicative[Ior[E, *]] =
+        Ior.catsDataMonadErrorForIor // See https://github.com/typelevel/cats/issues/3783
+
+      val applicative: Applicative[IorT[P.F, E, *]] = new Applicative[IorT[P.F, E, *]] {
+        def pure[A](a: A): IorT[P.F, E, A] = IorT.pure(a)(FA)
+        def ap[A, B](ff: IorT[P.F, E, A => B])(fa: IorT[P.F, E, A]): IorT[P.F, E, B] =
+          IorT(FA.map2(ff.value, fa.value)((f, a) => IorA.ap(f)(a)))
+      }
+
+      lazy val monad: Monad[IorT[M, E, *]] = {
+        implicit def underlyingMonadM: Monad[M] = P.monad
+        Monad[IorT[M, E, *]]
+      }
+    }
+
+  /**
+   * An alternative [[Parallel]] implementation which merges the semantics of
+   * the outer Parallel (the F[_] effect) with the effects of the inner
+   * one (the Ior). The inner Parallel has the semantics of [[Ior]]'s Parallel,
+   * while the outer has the semantics of parallel ''evaluation'' (in most cases).
+   * The default Parallel for [[IorT]], when the nested F also has a Parallel,
+   * is to strictly take the semantics of the nested F and to short-circuit any
+   * lefts (often, errors) in a left-to-right fashion, mirroring the semantics of
+   * [[Applicative]] on IorT. This instance is different in that it will not
+   * ''short-circuit'' but instead accumulate all lefts according to the supplied
+   * [[Semigroup]].
+   *
+   * {{{
+   * implicit val p: Parallel[IorT[IO, Chain[Error], *]] = IorT.accumulatingParallel
+   *
+   * val a = IorT(IO(Chain(error1).leftIor[Unit]))
+   * val b = IorT(IO(Chain(error2).leftIor[Unit]))
+   *
+   * (a, b).parTupled  // => IorT(IO(Chain(error1, error2).leftIor[Unit]))
+   * }}}
+   */
+  def accumulatingParallel[M[_], E](implicit
+    P: Parallel[M],
+    E: Semigroup[E]
+  ): Parallel.Aux[IorT[M, E, *], IorT[P.F, E, *]] { type Dummy } =
+    new Parallel[IorT[M, E, *]] {
+      type F[x] = IorT[P.F, E, x]
+      type Dummy // fix to make this one more specific than the catsDataParallelForIorTWithSequentialEffect, see https://github.com/typelevel/cats/pull/2335#issuecomment-408249775
```

Comment:
I think we can remove the `Dummy` types here, since they're just an implicit search workaround.

Reply:
Addressed 49fafb0